GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -555,21 +577,43 @@ public static ArgumentString GetAllChangedFilesCmd(bool excludeIgnoredFiles, Unt
             return args;
         }
 
-        public static ArgumentString MergeBranchCmd(string branch, bool allowFastForward, bool squash, bool noCommit, string strategy, bool allowUnrelatedHistories, string? mergeCommitFilePath, int? log)
+        public static ArgumentString MergeBranchCmd(string branch,
+                                                    bool allowFastForward,
+                                                    bool squash,
+                                                    bool noCommit,
+                                                    string strategy,
+                                                    bool allowUnrelatedHistories,
+                                                    string? mergeCommitFilePath,
+                                                    int? log,
+                                                    bool gpgSign = false,
+                                                    string? gpgKeyId = "",
```

Comment:
In `RebaseCmd` this argument defaults to `null`. Why is it different here?

Reply:
Good catch.  Yes it can match null.